spec/version_spec.rb

```diff
@@ -33,6 +46,33 @@ def self.supported?
         end
       end
 
+      RSpec.describe Builderator::Control::Version do
+        before(:context) do
+          SCM.unregister(Test)
+          SCM.register(NoHistory)
+        end
+
+        after(:context) do
+          SCM.unregister(NoHistory)
+          SCM.register(Test)
+        end
+
+        context 'current' do
+          it 'provides an error message if no version is found' do
+            version_file = Util.relative_path('VERSION')
+            version_file.delete if version_file.exist?
+
+            expect(SCM.tags.last).to be_nil
+            expect { Version.current }.to raise_error RuntimeError
```

Comment:
I'm pretty sure `expect { Version.current }.to raise_error` is sufficient, unless we really care about the error type.